### PR TITLE
Promises chaining

### PR DESCRIPTION
**Pull request by kiniggit**

### CODE REVIEW

File: 1-js/11-async/03-promise-chaining/article.md [hunk 1]
```diff
@@ -157,34 +139,34 @@ loadScript("/article/promise-chaining/one.js")
...
-This code can be made bit shorter with arrow functions:
+Esse código pode ser resumido utilizando arrow functions: 
 
 ```js run
 loadScript("/article/promise-chaining/one.js")
   .then(script => loadScript("/article/promise-chaining/two.js"))
   .then(script => loadScript("/article/promise-chaining/three.js"))
   .then(script => {
-    // scripts are loaded, we can use functions declared there
+    // scripts já carregadods, podemos utilizar funções declaradas neles
```
Comment (anapaulalemos) [inline 1-js/11-async/03-promise-chaining/article.md:157]: Minor typo: "já carregados" and not "já carregadods".

File: 1-js/11-async/03-promise-chaining/article.md [hunk 2]
```diff
@@ -199,21 +181,19 @@ loadScript("/article/promise-chaining/one.js").then(script1 => {
 });
```
Comment (anapaulalemos) [inline 1-js/11-async/03-promise-chaining/article.md:181]: You forgot to translate the comment on line 175:
` // this function has access to variables script1, script2 and script3`

File: 1-js/11-async/03-promise-chaining/01-then-vs-catch/solution.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
-The short answer is: **no, they are not the equal**:
+A resposta breve é: **não, eles não são iguais**:
 
-The difference is that if an error happens in `f1`, then it is handled by `.catch` here:
+A diferença é que se um erro ocorrer em `f1`, então ele será tratado pelo `.catch` neste caso:
```
Comment (anapaulalemos) [inline 1-js/11-async/03-promise-chaining/01-then-vs-catch/solution.md:3]: I think the word 'then' and the previous comma are not necessary in this case

**Comment by kiniggit** [comment]
Hey, Ana, thank you for your input!

**Pull request merged**